GitUI/UserControls/ToolStripEx.cs

```diff
@@ -18,6 +18,59 @@ public ToolStripEx()
             _gripButton = propGrip.GetValue(this) as ToolStripButton;
         }
 
+        protected override void OnItemAdded(ToolStripItemEventArgs e)
+        {
+            base.OnItemAdded(e);
+
+            if (e.Item is ToolStripDropDownItem item)
+            {
+                item.DropDownOpening += SplitButton_DropDownOpening;
+                item.DropDownClosed += SplitButton_DropDownClosed;
+            }
+        }
+
+        protected override void OnItemRemoved(ToolStripItemEventArgs e)
+        {
+            if (e.Item is ToolStripDropDownItem item)
+            {
+                item.DropDownOpening -= SplitButton_DropDownOpening;
+                item.DropDownClosed -= SplitButton_DropDownClosed;
+            }
+        }
+
+        private static void SuspendDrawing(Control control)
+        {
+            if (control != null)
```

Comment:
```suggestion
            if (control is not null)
```
is our coding style.

(If there was more code, rather:
```suggestion
            if (control is null)
            {
                return;
            }
```
)

Reply:
Methods are gone (inlined) so this code is gone too.